test/adoption_center/test_client.rb

```diff
@@ -0,0 +1,18 @@
+require 'minitest/autorun'
+require '../lib/client'
+include Client
```

Comment:
Hey @angela4713 good job!

I think you're requiring `require '../lib/client'` ans then including the module because Rails is not loading the module for you.@bf4's comment is totally on point and including the module in the global scope might not be a good idea.

On the other hand, given that the adoption center will be using OAuth to authenticate users via RubyGems.org I think this client will actually not be necessary and we can just use the `gem` gem to get the list of gems of any given user. I I'm sorry I pointed you in this direction, it's my bad.

Ping me on Slack and lets pair on creating a basic `test_helper` that make your life easier and to discuss how we can deal with authentication while OAuth is ready on RubyGems.org side.